docs/getting_started/tutorial.md

```diff
@@ -120,23 +116,23 @@ The `requirements.txt` file contains all the dependencies needed to run the proj
 By default, your database is SQLite. To match your database tables with your project's models, run the following command:
 
 ```sh
-python manage.py migrate
+uv run manage.py migrate
 ```
 
 This command ensures that the tables in your database match the models in your project. Every time you alter your model, then you must run the `python manage.py migrate` command to update the database. For example, if you add a field to a model, then you must run the command.
 
 ### Create an admin user
 
 ```sh
-python manage.py createsuperuser
+uv run manage.py createsuperuser
 ```
 
 This prompts you to create a new admin user account with full permissions. It's important to note that for security reasons, the password text won’t be visible while typing.
 
 ### Start the server
 
 ```sh
-python manage.py runserver
+uv run manage.py runserver
 ```
```

Comment:
None of the `python manage.py` commands need `uv run`, right? Wouldn't it work even with `uv` if you already activated the environment?

Reply:
Yes `uv run` could be omitted.